evm/vm/base.py

```diff
@@ -691,27 +687,23 @@ def validate_seal(self, header: BlockHeader) -> None:
             header.block_number, header.mining_hash,
             header.mix_hash, header.nonce, header.difficulty)
 
-    def validate_uncle(self, block, uncle):
+    def validate_uncle(self, block, uncle, uncle_parent):
```

Comment:
Maybe `niece` instead of `uncle_parent`? I had a hard time parsing `uncle_parent`, which seems to mean `grandparent` without digging further.

Reply:
I think uncles can come from even a few blocks back so it's not something we can anchor to the `block` (and even the term `uncle` is a misnomer as what we call uncles can actually be great-uncles (block -> parent -> great-parent -> uncle (aka grand-uncle)).

So this is meant to be the *parent of the uncle*.  No better names come to mind.

Reply:
Got it, thanks. I misread the validation. I also cannot think of a better name.